lib/async_hooks.js

```diff
@@ -148,6 +150,35 @@ function createHook(fns) {
 }
 
 
+function getActiveResources() {
+  const handles = process._getActiveHandles();
+  const reqs = process._getActiveRequests();
+
+  const timers = {};
+  for (const list of ObjectValues(internalTimers.timerLists)) {
```

Comment:
To avoid unsafe array iteration:
```suggestion
  ArrayPrototypeForEach(ObjectValues(internalTimers.timerLists), (list) => {
```